setup.py

```diff
@@ -14,6 +15,27 @@
 
 from setuptools import find_packages
 from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+
+class PyTest(TestCommand):
```

Comment:
Just an idea. :smiley: You don't have to integrate it now, but you can experiment it if it works.